README.md

```diff
@@ -13,6 +13,34 @@ The __filter__ package is generally provided as a folder with source (`.java`) a
 
 A sample set of images are provided in an __"images"__ directory.
 
+## Creating an executable jar file
+If there is no automated build environment set up, the jar file can be created using "jar."
```

Comment:
I appreciate the inclusion of this information!
It seems like it fits best _after_ the Gradle build instructions. (Even the opening sentence seems to make more sense in that position.)